src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/package.json

```diff
@@ -8,6 +8,8 @@
     "build": "cross-env NODE_ENV=production webpack --config webpack.config.production.js --progress",
     "build-develop": "cross-env NODE_ENV=development webpack --config webpack.config.development --progress --watch",
     "test": "jest",
```

Comment:
You could also execute `yarn run test --watch` to achieve this. Personally I would prefer to keep the scripts list as short as possible, but I dont have any objections to merge this either.

Reply:
Cool, I did not know you could do that. It definelty solves my usecase.

Still a separate script is much more discoverable, and serves as documentation the next time someone who does not know jest and npm script arguments wants to run the tests in watch mode.